lib/src/DynamicProto.ts

```diff
@@ -135,8 +140,13 @@ function _getObjProto(target:any) {
         }
 
         // target[Constructor] May break if the constructor has been changed or removed
-        let newProto = target[str__Proto] || target[Prototype] || (target[Constructor] ? target[Constructor][Prototype] : null);
+        let newProto = target[DynProtoPolyProto] || target[str__Proto] || target[Prototype] || (target[Constructor] ? target[Constructor][Prototype] : null);
         if(newProto) {
+            // Set the _dynProto so the hierarchy can lookup base classes correctly on IE8
+            if (!target[DynProtoPolyProto]) {
```

Comment:
Because IE8 doesn't support looking up the prototype of a prototype, we need to provide hints to allow base classes that are also defined using dynamicProto() to be looked up.

This issue only exists for nested classes of 2 or more. And this part of the code only runs on older browsers